From 9f70b94ebf18f52c115634642652830fa77f27a1 Mon Sep 17 00:00:00 2001
From: "Maxin B. John" <maxin.john@intel.com>
Date: Mon, 12 Jun 2017 16:52:39 +0300
Subject: [PATCH] connman.service: stop systemd-resolved when we use connman

Stop systemd-resolved service when we use connman as network manager.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Maxin B. John <maxin.john@intel.com>
---
 src/connman.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/connman.service.in b/src/connman.service.in
index 9f5c10f..dab48bc 100644
--- a/src/connman.service.in
+++ b/src/connman.service.in
@@ -6,6 +6,7 @@ RequiresMountsFor=@localstatedir@/lib/connman
 After=dbus.service network-pre.target systemd-sysusers.service
 Before=network.target multi-user.target shutdown.target
 Wants=network.target
+Conflicts=systemd-resolved.service
 
 [Service]
 Type=dbus
-- 
2.4.0

